### ISSUE TITLE
Update etcd

### ISSUE THREAD

**Issue by khenderick** [openvstorage/framework#546]
Update etcd to latest version. Then validate whether etcdctl still executes dirty reads.

```
root@ovs100:~# etcdctl --debug get /ovs/foo/bar
start to sync cluster using endpoints(http://127.0.0.1:4001,http://127.0.0.1:2379)
cURL Command: curl -X GET http://127.0.0.1:4001/v2/members
cURL Command: curl -X GET http://127.0.0.1:2379/v2/members
got endpoints(http://10.100.169.100:2379) after sync
Cluster-Endpoints: http://10.100.169.100:2379
cURL Command: curl -X GET http://10.100.169.100:2379/v2/keys/ovs/foo/bar?quorum=false&recursive=false&sorted=false
Error:  100: Key not found (/ovs/foo) [38]
root@ovs100:~# etcdctl --version
etcdctl version 2.2.3
root@ovs100:~# 
```

**Comment by domsj** [comment]
Quorum reads were added to etcdctl in this commit: https://github.com/coreos/etcd/commit/11f49a09605b7590fe1b3e7892ff8cc3c66dbe71
As you can see there it should be available as of v2.3.0.
Once you've updated etcdctl to this version we can make a new version of alba & arakoon which adds the --quorum flag to do consistent reads.

**Comment by khenderick** [comment]
@domsj I've just updated etcd to 2.3.5 on unstable. Can you check out the new `--quorum` option in etcdctl and if possible start using it?
